src/inspector/tracing_agent.cc

```diff
@@ -173,14 +173,25 @@ DispatchResponse TracingAgent::stop() {
 DispatchResponse TracingAgent::getCategories(
     std::unique_ptr<protocol::Array<String>>* categories) {
   *categories = Array<String>::create();
-  categories->get()->addItem("node");
-  categories->get()->addItem("node.async");
-  categories->get()->addItem("node.bootstrap");
-  categories->get()->addItem("node.fs.sync");
-  categories->get()->addItem("node.perf");
-  categories->get()->addItem("node.perf.usertiming");
-  categories->get()->addItem("node.perf.timerify");
-  categories->get()->addItem("v8");
+  protocol::Array<String>* categorie_list = categories->get();
```

Comment:
typo:
```suggestion
  protocol::Array<String>* categories_list = categories->get();
```
and add the `s` everywhere?

Reply:
Fixed. Thanks.